openmc/mesh.py

```diff
@@ -181,6 +183,21 @@ def __repr__(self):
         string += '{0: <16}{1}{2}\n'.format('\tPixels', '=\t', self._width)
         return string
 
+    def cell_generator(self):
+        """Generator function to traverse through every [i,j,k] index
+        of the mesh.
+
+        """
+        if len(self.dimension) == 2:
+            for x in range(self.dimension[0]):
+                for y in range(self.dimension[1]):
+                    yield [x + 1, y + 1, 1]
+        else:
+            for x in range(self.dimension[0]):
+                for y in range(self.dimension[1]):
+                    for z in range(self.dimension[2]):
+                        yield [x + 1, y + 1, z + 1]
```

Comment:
Is there are reason to use the `+ 1`'s here? One place where the output of this method is used, lines 285-293 of this file, `- 1`'s are used to negate the `+ 1`'s.